authority-portal-frontend/src/app/pages/registration-pages/organization-create-page/organization-create-page/organization-create-page.component.html

```diff
@@ -1,8 +1,11 @@
 <div class="relative h-screen">
   <!-- header -->
   <div
-    class="top-0 p-8 bg-white flex justify-start items-center cursor-pointer">
-    <img src="/assets/images/mds_logo_black.svg" alt="logo" />
+    class="top-0 p-8 bg-color-background flex justify-start items-center cursor-pointer">
```

Comment:
and here you use bg-color-background instead of bg-white?

Reply:
So funny thing about that. I changed the background color by adding it to .body which took care of most backgrounds.
Now some components didn't have `bg-white` but I wanted them to stay white to contrast the darker background.
But other components for some reason had `bg-white` applied despite not needing it because the background was already white? For some of them, I actually needed them to blend in with the background, thus the chaotic changes everywhere